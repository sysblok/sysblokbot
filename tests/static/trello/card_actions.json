[
    {
      "id": "5ebc8811bc56ff5809c1f833",
      "idMemberCreator": "56cd570f0c81722559a4abd7",
      "data": {
        "old": {
          "idList": "5e91d48a9e9d947b95fb0e00"
        },
        "card": {
          "idList": "list_1",
          "id": "1",
          "name": "Картографирование эпидемий",
          "idShort": 166,
          "shortLink": "9qBC9AWi"
        },
        "board": {
          "id": "5e91d48a9e9d947b95fb0df7",
          "name": "Редакция (тест)",
          "shortLink": "Kz1xBlTf"
        },
        "listBefore": {
          "id": "list_8",
          "name": "Готово для верстки (Выпускающему)"
        },
        "listAfter": {
          "id": "list_4",
          "name": "На редактуре на след.неделю (не забудь тег рубрики!)"
        }
      },
      "type": "updateCard",
      "date": "2020-05-13T23:51:45.420Z",
      "limits": {},
      "memberCreator": {
        "id": "56cd570f0c81722559a4abd7",
        "username": "alexeyqu",
        "activityBlocked": false,
        "avatarHash": "b8c99e483cd8b3ba037505feceac4483",
        "avatarUrl": "https://trello-members.s3.amazonaws.com/56cd570f0c81722559a4abd7/b8c99e483cd8b3ba037505feceac4483",
        "fullName": "Alexey Kulikov",
        "idMemberReferrer": null,
        "initials": "AK",
        "nonPublic": {},
        "nonPublicAvailable": false
      }
    },
    {
      "id": "5eb8727f1218f340bc659093",
      "idMemberCreator": "550c620696262ccd4a506b3f",
      "data": {
        "old": {
          "pos": 3994744307712
        },
        "card": {
          "pos": 3994744389632,
          "id": "1",
          "name": "Картографирование эпидемий",
          "idShort": 166,
          "shortLink": "9qBC9AWi"
        },
        "board": {
          "id": "5e91d48a9e9d947b95fb0df7",
          "name": "Редакция (тест)",
          "shortLink": "Kz1xBlTf"
        },
        "list": {
          "id": "list_8",
          "name": "Готово для верстки (Выпускающему)"
        }
      },
      "type": "updateCard",
      "date": "2020-05-10T21:30:39.132Z",
      "limits": {},
      "memberCreator": {
        "id": "550c620696262ccd4a506b3f",
        "username": "irinoise",
        "activityBlocked": false,
        "avatarHash": "f8da091665b4097042ec7eb1fe5de251",
        "avatarUrl": "https://trello-members.s3.amazonaws.com/550c620696262ccd4a506b3f/f8da091665b4097042ec7eb1fe5de251",
        "fullName": "Irina Shakhova",
        "idMemberReferrer": null,
        "initials": "IS",
        "nonPublic": {},
        "nonPublicAvailable": false
      }
    },
    {
      "id": "5eb8727f1218f340bc659091",
      "idMemberCreator": "550c620696262ccd4a506b3f",
      "data": {
        "old": {
          "idList": "5e91d48a9e9d947b95fb0dfc"
        },
        "card": {
          "idList": "5e91d48a9e9d947b95fb0e00",
          "id": "1",
          "name": "Картографирование эпидемий",
          "idShort": 166,
          "shortLink": "9qBC9AWi"
        },
        "board": {
          "id": "5e91d48a9e9d947b95fb0df7",
          "name": "Редакция (тест)",
          "shortLink": "Kz1xBlTf"
        },
        "listBefore": {
          "id": "list_4",
          "name": "На редактуре на след.неделю (не забудь тег рубрики!)"
        },
        "listAfter": {
          "id": "list_8",
          "name": "Готово для верстки (Выпускающему)"
        }
      },
      "type": "updateCard",
      "date": "2020-05-10T21:30:39.108Z",
      "limits": {},
      "memberCreator": {
        "id": "550c620696262ccd4a506b3f",
        "username": "irinoise",
        "activityBlocked": false,
        "avatarHash": "f8da091665b4097042ec7eb1fe5de251",
        "avatarUrl": "https://trello-members.s3.amazonaws.com/550c620696262ccd4a506b3f/f8da091665b4097042ec7eb1fe5de251",
        "fullName": "Irina Shakhova",
        "idMemberReferrer": null,
        "initials": "IS",
        "nonPublic": {},
        "nonPublicAvailable": false
      }
    }
]
