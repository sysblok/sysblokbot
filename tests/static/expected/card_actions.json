[
    {
      "id": "5ebc8811bc56ff5809c1f833",
      "data": {
        "card": {
          "shortLink": "9qBC9AWi"
        },
        "listBefore": {
          "id": "list_8",
          "name": "Готово для верстки (Выпускающему)"
        },
        "listAfter": {
          "id": "list_4",
          "name": "На редактуре на след.неделю (не забудь тег рубрики!)"
        }
      },
      "date": "2020-05-13T23:51:45.420Z"
    },
    {
      "id": "5eb8727f1218f340bc659093",
      "data": {
        "card": {
          "shortLink": "9qBC9AWi"
        },
        "listBefore": {
          "id": null,
          "name": null
        },
        "listAfter": {
          "id": null,
          "name": null
        }
      },
      "date": "2020-05-10T21:30:39.132Z"
    },
    {
      "id": "5eb8727f1218f340bc659091",
      "data": {
        "card": {
          "shortLink": "9qBC9AWi"
        },
        "listBefore": {
          "id": "list_4",
          "name": "На редактуре на след.неделю (не забудь тег рубрики!)"
        },
        "listAfter": {
          "id": "list_8",
          "name": "Готово для верстки (Выпускающему)"
        }
      },
      "date": "2020-05-10T21:30:39.108Z"
    }
]
